<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderWidget</class>
 <widget class="QWidget" name="OrderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Order widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_19">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_17">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page1">
        <layout class="QVBoxLayout" name="verticalLayout_21">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="3">
              <widget class="QGroupBox" name="groupBox_3">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="title">
                <string>Time</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QTimeEdit" name="timeEditTime"/>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="8">
              <widget class="QGroupBox" name="groupBox_9">
               <property name="title">
                <string>Cash</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_14">
                <item>
                 <widget class="SpinBoxDecimal" name="doubleSpinBoxCash">
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                  </property>
                  <property name="showGroupSeparator" stdset="0">
                   <bool>true</bool>
                  </property>
                  <property name="maximum">
                   <double>9999.989999999999782</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QGroupBox" name="groupBox_6">
               <property name="title">
                <string>Covers</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <widget class="QSpinBox" name="spinBoxCovers">
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="7">
              <widget class="QGroupBox" name="groupBox_8">
               <property name="title">
                <string>Discount</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_13">
                <item>
                 <widget class="SpinBoxDecimal" name="doubleSpinBoxDiscount">
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                  </property>
                  <property name="showGroupSeparator" stdset="0">
                   <bool>true</bool>
                  </property>
                  <property name="maximum">
                   <double>9999.989999999999782</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="2" colspan="5">
              <widget class="QGroupBox" name="groupBox_11">
               <property name="title">
                <string>Customer name</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_16">
                <item>
                 <widget class="QLineEdit" name="lineEditCustomerName"/>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QGroupBox" name="groupBox">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="title">
                <string>Number</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QSpinBox" name="spinBoxNumber">
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                  </property>
                  <property name="minimum">
                   <number>0</number>
                  </property>
                  <property name="maximum">
                   <number>99999</number>
                  </property>
                  <property name="value">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>Table</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <widget class="QLineEdit" name="lineEditTableNumber"/>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="7">
              <widget class="QGroupBox" name="groupBox_12">
               <property name="title">
                <string>Order status</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_18">
                <item>
                 <widget class="RelationalComboBox" name="comboBoxStatus"/>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="9">
              <widget class="QGroupBox" name="groupBox_10">
               <property name="title">
                <string>Change</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="SpinBoxDecimal" name="doubleSpinBoxChange">
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                  </property>
                  <property name="showGroupSeparator" stdset="0">
                   <bool>true</bool>
                  </property>
                  <property name="maximum">
                   <double>9999.989999999999782</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="8" colspan="2">
              <widget class="QGroupBox" name="groupBox_13">
               <property name="title">
                <string>Fullfillment date</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_20">
                <item>
                 <widget class="DateTimeEdit" name="dateTimeEditFullfillment">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBox_14">
               <property name="title">
                <string>Cash desk</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_15">
                <item>
                 <widget class="QLineEdit" name="lineEditCashDesk"/>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="6">
              <widget class="QGroupBox" name="groupBox_7">
               <property name="title">
                <string>Total amount</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_12">
                <item>
                 <widget class="SpinBoxDecimal" name="doubleSpinBoxTotalAmount">
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                  </property>
                  <property name="showGroupSeparator" stdset="0">
                   <bool>true</bool>
                  </property>
                  <property name="maximum">
                   <double>9999.989999999999782</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QGroupBox" name="groupBox_2">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="title">
                <string>Date</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QDateEdit" name="dateEditDate"/>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>Delivery</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="RelationalComboBox" name="comboBoxDelivery"/>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QGroupBox" name="groupBox_15">
               <property name="title">
                <string>Elect. paym.</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_22">
                <item alignment="Qt::AlignmentFlag::AlignHCenter">
                 <widget class="QCheckBox" name="checkBoxElectronicPayment">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Order details</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="EnhancedTableView" name="tableViewDetails"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Departments details</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="EnhancedTableView" name="tableViewDepartmentDetails"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Department notes and status</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="EnhancedTableView" name="tableViewDepartmentHeader"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page2">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="EnhancedTableView" name="tableView"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBoxReprint">
         <property name="title">
          <string>Current order reprint</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButtonPrint">
            <property name="text">
             <string>Print</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxPrintCustomerCopy">
            <property name="text">
             <string>Customer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxPrintCoverCopy">
            <property name="text">
             <string>Cover</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EnhancedTableView</class>
   <extends>QTableView</extends>
   <header>App.Widget.View</header>
  </customwidget>
  <customwidget>
   <class>RelationalComboBox</class>
   <extends>QComboBox</extends>
   <header>App.Widget.Control</header>
  </customwidget>
  <customwidget>
   <class>DateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>App.Widget.Control</header>
  </customwidget>
  <customwidget>
   <class>SpinBoxDecimal</class>
   <extends>QDoubleSpinBox</extends>
   <header>App.Widget.Control</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
